README.md

```diff
@@ -35,26 +44,69 @@ Env variables for configuration:
+In the directory `config/refbox` are some configurations which are used by default. You can find more [here](https://github.com/robocup-logistics/rcll-refbox/tree/master/cfg]) in the rcll-refbox repository.
+
+The startup utilizes docker-compose to spin up services and then sets up a screen session where the logs can be followed.
+The screen session is named `${RC_SCREEN_NAME}`, which defaults to `rcll`.
+The existing tabs in that session are always closed before re-launching.
+
+##### rc_stop
+This stops all services.
+
+##### rc_pull
+This pulls the newest version of all docker images.
+
+#### Other Commands
+
+The RCLL offers also some additional tools. For each, there are 3 commands:
+ - rc_start_<tool> - which launches the tool
+ - rc_stop_<tool> - which stops the tool
+ - rc_pull_<tool> - which updates the docker containers for that tool
+
+There are also options for each tool to be launched in the main commands above.
+
+##### refbox
+
+Used to just start the refbox and it's frontend without any other tools.
+See the `rc_start` section for configuration options.
+
+Note that the startup also checks for the existance of a MongoDB un `RC_MONGODB_URI`.
+
+##### mongodb
+
+The refbox requires a running MongoDB instance (any version compatible with the mongocxx-driver of the refbox).
+Per default, a new MongoDB 5.0 is started on startup. But it is also possible to use an existing instance. Note that the refbox needs to be configured separately to also use the instance.
+
+Env variables for configuration:
+- `RC_MONGODB_START` set to true if a mongodb instance should start with the main command, defaults to `true`
+- `RC_MONGODB_HOST` hostname or ip address where the instance should run on, defaults to `localhost`
+- `RC_MONGODB_PORT` port where the intance should run on, defaults to `27017`
+- `RC_MONGODB_URI` Final URI, defaults to` mongodb://${RC_MONGODB_HOST}:${RC_MONGODB_PORT}`
+
+##### mongodb_backend
+
+The refbox logs games into a database, the frontend of the refbox can display that data.
+In order to do so, a small backend is required that bridges between the database and the frontend.
+
+Env variables for configuration:
+- `RC_MONGODB_BACKEND_START` set to true if the mongodb backend should be started with the main command, defaults to `false`
+- `RC_MONGODB_BACKEND_DB_NAME` name of the database to use, defaults to `rcll`, which is the default name of the database the refbox uses.
+
+Note that the backend also utilizes the `RC_MONGODB_URI` variable from the mongodb component.
 
-#### rc_stop_refbox
-This stops the refbox (this command also calls: `rc_stop_mqtt_bridge`, so the mqtt bridge is also stopped)
-  
-#### rc_start_mqtt_bridge
+##### mqtt_bridge
 
 Note: The image is only available via ghcr.io, so you have to login in order to be able to pull it!
```

Comment:
I think it should now also be available without a login since we changed the settings from the organization a couple of months ago. I will test it from a different PC to confirm this.

Reply:
sure i also added it to quay, btw.